.github/workflows/build-scala-cli-template/action.yml

```
@@ -5,6 +5,8 @@ runs:
   steps:
     - name: Install CIRCT
       uses: ./.github/workflows/install-circt
+      with:
+        github-token: ${{ gitub.token }}
```

```suggestion
        github-token: ${{ github.token }}
```
This typo is breaking CI from running 😂